src/cargo-fmt/main.rs

```diff
@@ -215,7 +213,7 @@ fn print_usage_to_stderr(reason: &str) {
         .expect("failed to write to stderr");
 }
 
-#[derive(Debug, Clone, Copy, PartialEq)]
+#[derive(Debug, Clone, Copy, PartialEq, Eq)]
```

Comment:
warning: you are deriving `PartialEq` and can implement `Eq`
```
 --> src/lib.rs:1:30
  |
1 | #[derive(Debug, Clone, Copy, PartialEq)]
  |                              ^^^^^^^^^ help: consider deriving `Eq` as well: `PartialEq, Eq`
  |
```
  = note: `#[warn(clippy::derive_partial_eq_without_eq)]` on by default
  = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#derive_partial_eq_without_eq